docs/source/model_design/vivarium_model_components/risk_factors/risk_mediation/index.rst

```diff
@@ -44,214 +45,505 @@ but that effect is (completely) mediated by another risk factor R2.
+These are also called "log-linear" effects, since for each unit change in the independent
+variable, there is a multiplicative factor applied to the dependent variable --
+which is equivalent to a constant change in its *logarithm*.
 
-.. image:: bmi-sbp-mediation.drawio.svg
+Combining the effects of multiple pathways that each have RRs is straightforward.
+We simply multiply the RRs of each pathway to determine the RR of the total effect.
+Going back to this simple diagram:
 
-BMI here is body mass index, SBP is systolic blood pressure, and IHD
-is ischemic heart disease.
-We see that BMI has an effect on IHD mediated by SBP, but also has
-a direct effect.
+.. image:: mediation-with-direct-color.drawio.svg
 
-Imagine we found studies describing the effects in this diagram
-by reporting relative risks (RRs).
-To inform the direct BMI→IHD arrow, we'd need the RR of BMI
-on IHD to be adjusted for SBP, in order to isolate the direct effect.
-If we used an RR of BMI on IHD that was not adjusted for SBP, the mediated
-part of the effect would be double-counted in our model: once in the direct
-path and once in the mediated path.
-This logic extends naturally to the situation where
-there are multiple mediated pathways in addition to a direct effect:
-for the direct effect, we need an RR adjusted for **all** mediated pathways.
+we can say that
 
-As always when we use risk effects, whether or not mediation is involved,
-the effects should be causal; they should exclude :ref:`confounding <confounding>`.
-In an observational study, this means they should be adjusted for known
-confounders/common causes (factors affecting both our risk and our outcome),
-**in addition** to adjusting for the modeled mediator when calculating the direct RR.
-Crucially, in this case, the RR of SBP→IHD **must be adjusted for BMI** since
-that is a confounder of this effect.
+.. math::
 
-Note that while the ideal would be evidence from a randomized controlled trial,
-therefore controlling for *all* confounders (known and unknown), that frequently isn't possible,
-including in this example:
-there isn't a way to randomly assign people BMIs or SBPs since these variables
-are not directly manipulable.
+  RR_\text{r→o,total} = RR_\text{r→o,direct} \times RR_\text{r1→r2→o}
 
-GBD-modeled mediation
-+++++++++++++++++++++
+This equation can be used in either direction, depending on what we have
+data about.
+If the total effect is known, we can split it;
+if it is not known, we can infer what it is by combining
+the effects of each pathway.
+
+Another case in which combining and splitting are straightforward
+is when all the pathways have additive, or "linear," effects.
+These can simply be added to obtain the total effect.
+For example, if a 1 unit increase in R1 in the diagram above caused a 2.2 unit increase in O
+through R2, and also caused a 1.3 unit increase in O directly, the total
+effect would be :math:`2.2 + 1.3 = 3.5` units of increase in O per unit increase in R1.
```

Comment:
Do you think it's worth noting that this is the case as long as there is no interaction between R1 and R2 and their effects on O?

Reply:
I called out above that this assumption is made throughout the document, but I think it's worth noting it again in this section about splitting! The whole idea of "splitting" doesn't really work when there is effect modification.